source/app.d

```diff
@@ -38,6 +42,8 @@ void startServer(HTTPServerSettings settings)
     settings.bindAddresses = ["0.0.0.0"];
     settings.options = HTTPServerOption.defaults & ~HTTPServerOption.parseJsonBody;
 
+    lastFullPRCheck = SysTime(Date(0, 1, 1));
```

Comment:
SysTime.init is zero and won't overflow, both SysTime and Duration use long hnsecs internally.